guides/common/modules/proc_creating-custom-provisioning-hooks.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Creating_Custom_Provisioning_Hooks_{context}"]
+= Creating Custom Provisioning Hooks
+
+A custom provisioning snippets allows you to execute custom code during host provisioning.
+You can run code before and/or after the provisioning process.
+
```

Comment:
Please add a sentence like:
Depending on the original template, multiple custom snippet hooks exists which can be used to have custom snippets. Check the original template first to know, which custom snippet can be used.